zerver/webhooks/front/view.py

```diff
@@ -16,8 +16,8 @@ def get_message_data(payload: WildValue) -> Tuple[str, str, str, str]:
     link = "https://app.frontapp.com/open/" + payload["target"]["data"]["id"].tame(check_string)
     outbox = payload["conversation"]["recipient"]["handle"].tame(check_string)
     inbox = payload["source"]["data"][0]["address"].tame(check_string)
-    subject = payload["conversation"]["subject"].tame(check_string)
-    return link, outbox, inbox, subject
+    topic = payload["conversation"]["subject"].tame(check_string)
+    return link, outbox, inbox, topic
```

Comment:
These are also not required as the meaning of "subject" is different in this case. It refers to the mail received by the service, i.e. [Front](https://front.com/).